examples/vpc-scenario-2/main.tf

```diff
@@ -60,24 +70,25 @@ module "ubuntu-xenial-ami" {
   release = "14.04"
 }
 
-resource "aws_key_pair" "main" {
-  key_name   = "${var.name}"
-  public_key = "${file(var.ssh_pubkey)}"
+module "vpc-sg" {
```

Comment:
We usually name the SG based on the primary role of the hosts the SG is attached to. So if we have an ELB with a SG, the name of the SG will have elb in it. Here is my recommendation: create 2 SG with the base SG module, one for ELB and one for the web instance, and then attach rules to each of those SG to match what was in place before.

Reply:
@ketzacoatl The "vpc-sg" module's ID is used by the "public-ssh-sg" and the "open-egress-sg" as their `security_group_id`.  Should I use the newly created security group for ELB or the security group for the web instance as the `security_group_id` (since the "vpc-sg" module will not be there)?